fixtures/proc-macro/src/lib.rs

```diff
@@ -383,6 +387,8 @@ pub struct RecordWithDefaults {
     opt_vec: Option<Vec<bool>>,
     #[uniffi(default = Some(42))]
     opt_integer: Option<i32>,
+    #[uniffi(default = 42)]
```

Comment:
This is great, thanks! Unfortunately this doesn't correctly support `#[uniffi(default)]` for Python - could you please add a test case for this scenario (along with the test case you added for an explicit value) and look into supporting it for Python? Not supporting it for Python isn't necessarily a show-stopper, but it would make the docs for default values a little unfortunate - and speaking of which, it would also be great to get some docs for this in the manual (the docs could be fairly terse though - I've a PR up which improves the docs at https://github.com/mozilla/uniffi-rs/pull/2599 and I'll incorporate your docs into that). A CHANGELOG entry would also be great, where you can link to this PR and get all the glory :)

Thanks!

Reply:
Updated. Please take a look.